src/constants/settings.ts

```diff
@@ -57,6 +57,7 @@ export const settings = {
   skippedArtists: "skippedArtists",
   theme: "theme",
   trayIcon: "trayIcon",
+  staticWindowTitle: "staticWindowTitle",
```

Comment:
Any particular reason you've added it after `trayIcon` instead of following the alphabet like the others haha?

Reply:
Used `trayIcon` as an example and decided to put it below it since in the Settings window it will be the next option available. Also didn't notice that there was an alphabetic order hahahah, fixed on last commit on every file.

Reply:
haha no worries, it was just that it's right below where it should be :)